### ISSUE TITLE
The given key was not present in the dictionary when running sample app

### ISSUE THREAD

**Issue by peterblazejewicz** [mawax/InfoBridge.SuperLinq#1]
Hi,
I believe I've configured the sample correctly (local instance of SO, session is established) when default query in constructed:
```cs
var r = new Queryable<Person>().Where(x => x.ContactId == 2).ToList();
```
(or with samples from SO blog posts) I always receive an error:

```text
{"The given key was not present in the dictionary." System.Exception{System.Collections.Generic.KeyNotFoundException}
```

```
at System.Collections.Generic.Dictionary`2.get_Item(TKey key)\r\n   
at SuperOffice.Util.TimeConverter.GetTimeZoneDataById(Int32 timeZoneId) 
in c:\\Builds\\N1\\Seven\\Rel80_C\\Sources\\Server\\Source\\SoCore\\Util\\TimeConverter.cs:line 569\r\n   
at InfoBridge.SuperLinq.Core.Projection.DateTimeConverter.GetTenantDataBaseTimeZone() 
in C:\\dev\\InfoBridge.SuperLinq\\src\\InfoBridge.SuperLinq.Core\\Projection\\DateTimeConverter.cs:line 40\r\n   
at InfoBridge.SuperLinq.Core.Projection.DateTimeConverter.ConvertFromTimeZone(DateTime fromConvertTimeUTC) in
C:\\dev\\InfoBridge.SuperLinq\\src\\InfoBridge.SuperLinq.Core\\Projection\\DateTimeConverter.cs:line 28\r\n   
at InfoBridge.SuperLinq.Core.Archives.ArchiveManager`1.DoTypeSpecificConversion(ArchiveExecutionContext executionContext, ColumnInfo column, Object parsedValue) 
in C:\\dev\\InfoBridge.SuperLinq\\src\\InfoBridge.SuperLinq.Core\\Archives\\ArchiveManager.cs:line 134\r\n   
at InfoBridge.SuperLinq.Core.Archives.ArchiveManager`1.ParseResult(ArchiveExecutionContext executionContext, IList`1 list)
in C:\\dev\\InfoBridge.SuperLinq\\src\\InfoBridge.SuperLinq.Core\\Archives\\ArchiveManager.cs:line 112\r\n   
at InfoBridge.SuperLinq.Core.Archives.ArchiveManager`1.InnerDoQuery(RestrictionBuilderBase restrictionBuilder, OrderByBuilder orderByBuilder, Int32 max, Int32 skip) 
in C:\\dev\\InfoBridge.SuperLinq\\src\\InfoBridge.SuperLinq.Core\\Archives\\ArchiveManager.cs:line 68\r\n   
at InfoBridge.SuperLinq.Core.Archives.ArchiveManager`1.DoQuery(RestrictionBuilderBase restrictionBuilder, OrderByBuilder orderByBuilder, Int32 noItems, Int32 skip)
in C:\\dev\\InfoBridge.SuperLinq\\src\\InfoBridge.SuperLinq.Core\\Archives\\ArchiveManager.cs:line 42\r\n   
at InfoBridge.SuperLinq.Core.LinqProvider.QueryExecutor`1.ExecuteCollection[N](QueryModel queryModel) 
in C:\\dev\\InfoBridge.SuperLinq\\src\\InfoBridge.SuperLinq.Core\\LinqProvider\\QueryExecutor.cs:line 25\r\n  
at Remotion.Linq.Clauses.StreamedData.StreamedSequenceInfo.ExecuteCollectionQueryModel[T](QueryModel queryModel, IQueryExecutor executor)\r\n   
at Remotion.Linq.Clauses.StreamedData.StreamedSequenceInfo.ExecuteQueryModel(QueryModel queryModel, IQueryExecutor executor)\r\n   
at Remotion.Linq.QueryModel.Execute(IQueryExecutor executor)\r\n   
at Remotion.Linq.QueryProviderBase.Execute(Expression expression)\r\n   
at Remotion.Linq.QueryProviderBase.System.Linq.IQueryProvider.Execute[TResult](Expression expression)\r\n   
at Remotion.Linq.QueryableBase`1.GetEnumerator()\r\n   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)\r\n   
at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)\r\n   
at InfoBridge.SuperLinq.TestApp.Program.Main(String[] args) 
in C:\\dev\\InfoBridge.SuperLinq\\src\\InfoBridge.SuperLinq.TestApp\\Program.cs:line 26
```

I'm not acustomed with SO context relation to timezone settings - that is somehow prominent in stack trace

Thanks!

**Comment by mawax** [comment]
Hi Peter,

It's trying to convert a datetime to utc using the SuperOffice time zone helpers. By default this is turned off but I think there is an issue with that so it seems to be always enabled. The quick way to solve it for you is to enable time zones in your SuperOffice tenant, but I'll work on a proper fix.

**Comment by mawax** [comment]
Issue is solved in 1.0.1 which also has been published to nuget
